org.vassalengine.vassal.yml

```diff
@@ -0,0 +1,52 @@
+id: org.vassalengine.vassal
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+command: vassal
+rename-desktop-file: vassal.desktop
+rename-icon: VASSAL
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
why does it need home?

Reply:
Vassal doesn't have a standard location for that, instead allowing the user to choose. It also does not use the filechooser portal.

<img alt="Screenshot" src="https://github.com/user-attachments/assets/af2d0ed1-4c5a-488b-b28a-3496ef4d521e" />

Reply:
I agree that would be useful. Unfortunately, it requires a significant refactoring effort. For now, I hope we can keep the current behavior, but we will take this into consideration for Vassal 4.

Reply:
the stopgap solution will be to default to a standard location while still letting the user to choose.

the manifest permission can reflect the default status, while those wanting to choose can override the permission